source/templates.tex

```diff
@@ -1338,9 +1338,16 @@
 const char p[] = "Vivisectionist";
 X<int,p> x2;                    // OK
 
-class A { constexpr A(const char*) {} };
+class A {
+  constexpr A(const char*) {}
+  auto operator<=>(A, A) = default;
+};
+
+template<A value> class Y {
+  @\commentellip@
+};
 
-X<A, "Pyrophoricity"> x3;       // OK, string literal is a constructor argument to A
+Y<"Pyrophoricity"> x3;       // OK, string literal is a constructor argument to A
```

Comment:
What's the rationale for the change from `X` to `Y` here?

Reply:
Never mind, I see it now. Looks like a change in P0732R2 was missing markup: I think we were supposed to change the rest of the example to:
```
template<class T, T p> class X {
/* ... */
};
X<const char*, "Studebaker"> x1; // error: string literal as template-argument
const char p[] = "Vivisectionist";
X<const char*,p> x2; // OK
```